e2e/lvmtest/lvmtest.go

```diff
@@ -7,8 +7,10 @@ import (
 	"os/exec"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/gluster/glusterd2/glusterd2/snapshot/lvm"
+	config "github.com/spf13/viper"
```

Comment:
Have an empty line before importing viper. The convention for imports is to group stdlib packages first, followed by project packages, and then external packages.

Reply:
Changed in later patchsets